shim.js

```diff
@@ -11,6 +11,8 @@ const storage = new Storage({
 });
 
 global.storage = storage;
+global.android = Platform.OS === 'android';
+global.ios = Platform.OS === 'ios';
 
```

Comment:
This is genius! ❤️ 

Should we rename it to something less generic? I'm afraid any dep. could somehow change the value on runtime and that could be a problem